package.json

```diff
@@ -5,7 +5,6 @@
     "homepage": "http://dev.splunk.com",
     "main": "index.js",
     "directories": {
-        "example": "examples",
```

Comment:
Did we also do an audit of the dependencies and devDependencies after removing examples? I wonder if there are libraries only being used in the examples that we can remove. That can be done as a followup to this PR also I don't want to block on that.